packages/jackett/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1411"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="77d7912835de603708188ffa1fcb9006f1304efc50c286dd0c5d0ac1ffe80184"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=("jackett" 'exec ${TERMUX_PREFIX}/bin/dotnet ${TERMUX_PREFIX}/lib/jackett/jackett.dll --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1')
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
+TERMUX_PKG_RM_AFTER_INSTALL="
+lib/jackett/libMono.Unix.so
```

Comment:
I am rejecting this. If this lib is published as part of the program, it is essential to the running of the program. Removing it will cause problem. If the check fails, then is should be manually recompiled.

Reply:
Technically DllNotFoundException error can still happen when Check For Updates but I see you disabled update so I wont mind
If preferable build the correct libMono.Unix.so manually